modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/classloader/DefaultArtifactClassLoaderResolver.java

```diff
@@ -213,9 +210,7 @@ public MuleDeployableArtifactClassLoader createApplicationClassLoader(Applicatio
 
     regionClassLoader.addClassLoader(appClassLoader, artifactClassLoaderFilter);
 
-    List<ArtifactPluginDescriptor> artifactPluginDescriptors =
-        PluginsDependenciesProcessor.removeExportedPackagesAlreadyExportedByTransitiveDependencies(PluginsDependenciesProcessor
-            .process(descriptor.getPlugins(), false, List::add));
+    List<ArtifactPluginDescriptor> artifactPluginDescriptors = process(descriptor.getPlugins(), false, List::add);
```

Comment:
Keeping this because although plugins have been already ordered, they are in a `Set`, so here we guarantee the ordered list needed for the class loaders creation.

Reply:
add this as a comment in the code